static_src/app.jsx

```diff
@@ -53,22 +56,29 @@ export default class App extends React.Component {
           </a>
         </div>
       </nav>
-      <div className="container-fluid">
-        <div className="row">
-          <nav className={ sidebarClasses }>
-            { sidebar }
-          </nav>
-          <main className={ mainClasses }>
-            { content }
-          </main>
+      <div className={ titleBarStyles['title_bar'] }>
+        <div class="nav_toggle">
+          <i class="fa fa-bars nav_toggle-icon"></i>
+          <div class="icon-reorder tooltips" data-original-title="Toggle Navigation" data-placement="bottom">
```

Comment:
Ya, apparently data attributes are really the only valid HTML and JSX attributes.

[Relevant JSX documentation](https://facebook.github.io/react/docs/jsx-gotchas.html#custom-html-attributes)